src/lib.rs

```diff
@@ -463,6 +467,113 @@ impl<'a> From<LanguageTag<String>> for LanguageTag<Cow<'a, str>> {
     }
 }
 
+#[cfg(feature = "serde")]
+impl From<SerdeLanguageTag> for LanguageTag<String> {
+    fn from(slt: SerdeLanguageTag) -> Self {
+        slt.lang_tag
+    }
+}
+
+impl<T> Into<String> for LanguageTag<T> where T: Deref<Target = str> {
+    fn into(self) -> String {
+        self.as_str().to_string()
+    }
+}
+
+impl TryFrom<String> for LanguageTag<String> {
+    type Error = LanguageTagParseError;
+
+    fn try_from(value: String) -> Result<Self, Self::Error> {
+        LanguageTag::from_str(value.as_ref())
+    }
+}
+
+// #[doc(cfg(feature = "serde"))] wait for this to be stable
+#[cfg(feature = "serde")]
+/// Language Tag Type that is infallable when used with `serde`'s [decorators](https://serde.rs/container-attrs.html#from). This is accomplished
+/// by having a default trait that auto-resolves to `en-US`. You must have crate feature `serde` enabled to use this.
+///
+/// Example:
+/// ```
+/// use oxilangtag::SerdeLanguageTag;
+/// use serde::{Serialize, Deserialize};
+///
+/// pub struct ContainerStruct {
+///     #[serde(into = "String", from = "String")]
+///     language: SerdeLanguageTag
+/// }
+/// ```
+///
+/// use oxilangtag::LanguageTag;
+/// See [`LanguageTag`] for details.
+#[derive(Clone, Debug)]
+pub struct SerdeLanguageTag {
+    lang_tag: LanguageTag<String>
+}
+
+#[cfg(feature = "serde")]
+impl Deref for SerdeLanguageTag {
+    type Target = LanguageTag<String>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.lang_tag
+    }
+}
+
+#[cfg(feature = "serde")]
+impl Default for SerdeLanguageTag {
+    fn default() -> Self {
+        match LanguageTag::parse("en-US") {
```

Comment:
The "und" (undefined) language tag might be better.

Reply:
Changed, Thanks!